readme.md

```diff
@@ -401,18 +377,70 @@ With everything else we've seen, Javascript executes one line of code, then when
 
 However, some operations in Javascript are **asynchronous**, meaning Javascript goes on to the next line of code without waiting for the previous line to complete.
 
-This is limited mostly to timing functions, and operations where Javascript is loading data from some other website.
-
-### Why doesn't Javascript wait for these operations to complete before going to the next line of code?
+#### Why doesn't Javascript wait for these operations to complete before going to the next line of code?
 
 Because otherwise the webpage would just "hang" until the operation completes. The browser can't do anything while Javascript is actively running. We've seen this when we've encountered infinite `while` loops. Asynchronicity is a way of preventing the computer from freezing.
 
 This risk is greatest when Javascript is making requests to other webpages. There's no way of knowing how long the request will take to complete. It could be near-instant, but if the target server is having a bad day, it could take who-knows-how-long. You don't want the operability of your computer to be at the mercy of some random computer somewhere else.
 
 In this small app we made, anything we want to be sure happens **after** those 5 seconds of computing should go inside the callback of the `setTimeout`. This way, we can be certain that it will run only when the 5 seconds are up.
 
+## You Do: [TimerJS](https://github.com/ga-wdi-exercises/timer_js) (40 minutes / 1:20)
+
+-------
+
+# Additional Topics
+
+### Event Defaults (10 minutes) (Bonus)
+
+Back to our in-class code. In `index.html`, replace your button with a link to Google...
+
+```html
+<body>
+  <a href="http://google.com">Click me!</a>
+</body>
+```
+
+Now, add an event listener to that link that brings up a `prompt` box, asking the user if they want to go to Google...
+
+```js
+var link = $("a");
+var handleClickEvent = function(evt){
+  var input = prompt("You sure you want to go to Google?");
+}
+link.on("click", handleClickEvent);
+```
 
-### The Event Object (4:35 - 4:40, 5 minutes)
+The problem is we don't know how to stop them from going to Google! They go anyway, whether they hit "OK" or "Cancel".
+
+Some elements have a default action they perform. In the case of `<a>`, that action is going to another webpage. You can prevent that default action with an Event method called `preventDefault`.
+
+```js
+var link = $("a")
+var handleClickEvent = function(e){
+  e.preventDefault();
+  prompt("You sure you want to go to Google?")
```

Comment:
maybe instead of go to google, you could incorporate something people see in the wild? e.g. Audio is currently playing, are you sure you want to leave?